components/PackagesBox.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { StyleSheet, View, Dimensions, Image, TouchableOpacity, Text} from 'react-native';
+import PrimaryButton from './../components/PrimaryButton';
+
+export default class PackagesBox extends React.Component {
+    static navigationOptions = {
+        header: null,
+        };
+        
+    constructor(props) {
+        super(props);
+        this.state = {
+            expandBox: false,
+        } 
+    };
+    expand = () => {
+        this.setState (
+            {expandBox: !this.state.expandBox} 
+          )
+        
+    } 
+    onPress = (item) => {
+        this.props.navigation.navigate('RequestStatus', {
+            confirm: item.confirmedEmail,
+            item: item,
+      });
+    }
+
+
+  render() {
+    if(!this.state.expandBox) {
```

Comment:
You shouldn't have two return options in the `render` function. Instead, you should have a function running inside the `return` statement that has the `if`/`else` block